# Service that can be used to get the current control type used by the
# 'panda_control_server'. Options are: 'joint_control' and 'joint_group_control'

bool verbose
---
string control_type